src/GitVersionCore.Tests/VariableProviderTests.cs

```diff
@@ -152,4 +152,29 @@ public void ProvidesVariablesInContinuousDeploymentModeForStableWhenCurrentCommi
 
         JsonOutputFormatter.ToJson(vars).ShouldMatchApproved(c => c.SubFolder("Approved"));
     }
+
+    [Test]
+    [Category("NoMono")]
+    [Description("Won't run on Mono due to source information not being available for ShouldMatchApproved.")]
```

Comment:
I can't see that `ShouldMatchApproved()` is used in the test, so the `Category` and `Description` attributes can safely be removed, no?